public/input_text_load_data.js

```diff
@@ -119,6 +146,43 @@ function loadData(collection) {
   });
 }
 
+// Changes XML to JSON
```

Comment:
Are you just trying to extract the temporal ranges from the XML and shove that into a JSON structure to pass downstream and the rest of the document is opaque?